sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClientOptions.cs

```diff
@@ -162,24 +80,31 @@ public object GetOverrideObject<T>(Func<object> objectConstructor)
             return _overrides.GetOrAdd(typeof(T), objectConstructor());
         }
 
-        // Will be removed like AddPolicy when we move to azure core
-        private void Copy(ArmClientOptions other)
+        private static void CopyPolicies(ClientOptions source, ClientOptions dest)
         {
-            Transport = other.Transport;
-            foreach (var pol in other.PerCallPolicies)
+            var perCallPoliciesProperty = source.GetType().GetProperty("PerCallPolicies", BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.GetProperty);
```

Comment:
We should remove this before GA.

Reply:
https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5849